aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -1296,12 +1522,18 @@ class AztecText : EditText, TextWatcher {
         consumeEditEvent = true
     }
 
-
     fun enableTextChangedListener() {
         consumeEditEvent = false
     }
 
     fun isTextChangedListenerDisabled(): Boolean {
         return consumeEditEvent
     }
+
+    fun refreshText() {
+        val selStart = selectionStart
+        val selEnd = selectionEnd
+        text = editableText
```

Comment:
I think we should probably call `disableTextChangedListener()` before resetting a text, just in case.